son/content/drivers_of_social_mobility/research_and_development_environment/business_spending_on_research_and_development/3.0/DR52-3.0-business-spending-on-research-and-development--by-year--table-format.csv

```diff
@@ -0,0 +1,11 @@
+"Year","Business expenditure on R&D (£billions)"
+"2023",49.973
+"2022",51.553
+"2021",53.359
+"2020",49.731
+"2019",50.242
+"2018",49.981
+"2017",46.48
+"2016",45.037
+"2015",42.869
+"2014",41.049
```

Comment:
In SON24, we have data from 2007 to 2018, but now we only have data from 2014.
Any idea why we're missing the 2007-2014 data?

Reply:
Not sure, I'll check with SMC!